.github/workflows/cherryPick.yml

```diff
@@ -69,6 +69,23 @@ jobs:
           ref: staging
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Decrypt Botify GPG key
+        if: github.actor == 'OSBotify'
+        run: cd .github/workflows && gpg --quiet --batch --yes --decrypt --passphrase="$LARGE_SECRET_PASSPHRASE" --output OSBotify-private-key.asc OSBotify-private-key.asc.gpg
+        env:
+          LARGE_SECRET_PASSPHRASE: ${{ secrets.LARGE_SECRET_PASSPHRASE }}
+
+      - name: Import Botify GPG Key
+        if: github.actor == 'OSBotify'
+        run: cd .github/workflows && gpg --import OSBotify-private-key.asc
+
+      - name: Set up git for Botify
+        if: github.actor == 'OSBotify'
+        run: |
+          git config user.signingkey DBF63700F60F5530
```

Comment:
Does this `signingkey` need to be stored as an environment variable?

Reply:
Nope, it's a public key ID and exposed by the GitHub UI when you click on `Verified`

<img width="302" alt="Screen Shot 2021-11-19 at 2 58 40 PM" src="https://user-images.githubusercontent.com/2838819/142696844-9acaed3a-0a76-4cd4-b6da-fce1a10f9347.png">